Client-lib explorer can panic on polling fallback when poll interval is not set

## Bug

`client-lib` mempool explorer can panic when address tracking falls back from websocket tracking to polling and no `WithPollInterval` option was provided.

Panic:

```text
panic: non-positive interval for NewTicker
```

### Root cause
WithPollInterval documents a default of 10 seconds, but NewExplorer initializes options with a zero-value explorerSvc:

`svcOpts := &explorerSvc{}`
So if callers enable tracking with WithTracker(true) but do not pass WithPollInterval, pollInterval remains 0.

If websocket connection setup fails, startTracking falls back to polling:

`e.trackWithPolling(ctx)
`Then polling starts a ticker with the zero interval:

`ticker := time.NewTicker(e.pollInterval)
`which panics.

### Trigger
This happens when:
1. Explorer tracking is enabled with WithTracker(true).
2. Caller does not pass WithPollInterval.
3. Websocket setup fails or no websocket connections are available.
4. Explorer falls back to polling.
5. time.NewTicker(0) panics

--- a/pkg/client-lib/explorer/mempool/explorer.go
+++ b/pkg/client-lib/explorer/mempool/explorer.go
@@ -61,9 +61,10 @@ import (
 )
 
 const (
-	BitcoinExplorer = "bitcoin"
-	pongInterval    = 60 * time.Second
-	pingInterval    = (pongInterval * 9) / 10
+	BitcoinExplorer     = "bitcoin"
+	defaultPollInterval = 10 * time.Second
+	pongInterval        = 60 * time.Second
+	pingInterval        = (pongInterval * 9) / 10
 )
 
 var (
@@ -117,7 +118,9 @@ func NewExplorer(baseUrl string, net arklib.Network, opts ...Option) (explorer.E
 		return nil, fmt.Errorf("invalid base url: %s", err)
 	}
 
-	svcOpts := &explorerSvc{}
+	svcOpts := &explorerSvc{
+		pollInterval: defaultPollInterval,
+	}
 	for _, opt := range opts {
 		opt(svcOpts)
 	}
@@ -129,6 +132,9 @@ func NewExplorer(baseUrl string, net arklib.Network, opts ...Option) (explorer.E
 			net:        net,
 			noTracking: svcOpts.noTracking,
 		}, nil
+	}
+	if svcOpts.pollInterval <= 0 {
+		return nil, fmt.Errorf("poll interval must be positive")
 	}
 
 	svc := &explorerSvc{

--- a/pkg/client-lib/explorer/mempool/explorer.go
+++ b/pkg/client-lib/explorer/mempool/explorer.go
@@ -69,11 +69,11 @@ const (
 
 var (
 	defaultExplorerUrls = utils.SupportedType[string]{
-		arklib.Bitcoin.Name:        "https://mempool.space/api",
+		arklib.Bitcoin.Name:        "https://mempool.arkade.sh/api",
 		arklib.BitcoinTestNet.Name: "https://mempool.space/testnet/api",
 		//arklib.BitcoinTestNet4.Name: "https://mempool.space/testnet4/api", //TODO uncomment once supported
-		arklib.BitcoinSigNet.Name:    "https://mempool.space/signet/api",
-		arklib.BitcoinMutinyNet.Name: "https://mutinynet.com/api",
+		arklib.BitcoinSigNet.Name:    "https://mempool.signet.arkade.sh/api",
+		arklib.BitcoinMutinyNet.Name: "https://mempool.mutinynet.arkade.sh/api",
 		arklib.BitcoinRegTest.Name:   "http://127.0.0.1:3000",
 	}
 )

--- a/pkg/client-lib/explorer/service_test.go
+++ b/pkg/client-lib/explorer/service_test.go
@@ -1093,6 +1093,38 @@ func TestStopClearsSubscriptions(t *testing.T) {
 	})
 }
 
+func TestNewExplorerPollInterval(t *testing.T) {
+	t.Run("uses default interval on websocket fallback", func(t *testing.T) {
+		//The test server is just a plain HTTP test server.
+		//It does not register /v1/ws, so websocket setup fails and the explorer falls back to polling.
+		ts := newTestServer(t)
+
+		svc, err := mempool.NewExplorer(
+			ts.URL, arklib.Bitcoin, mempool.WithTracker(true),
+		)
+		require.NoError(t, err)
+
+		require.NotPanics(t, func() { svc.Start() })
+		t.Cleanup(svc.Stop)
+		time.Sleep(25 * time.Millisecond)
+
+		require.Equal(t, 0, svc.GetConnectionCount())
+	})
+
+	t.Run("rejects non-positive interval", func(t *testing.T) {
+		ts := newTestServer(t)
+
+		for _, interval := range []time.Duration{0, -time.Second} {
+			_, err := mempool.NewExplorer(
+				ts.URL, arklib.Bitcoin,
+				mempool.WithTracker(true),
+				mempool.WithPollInterval(interval),
+			)
+			require.ErrorContains(t, err, "poll interval must be positive")
+		}
+	})
+}
+
 func TestStartIsIdempotent(t *testing.T) {
 	t.Run("valid", func(t *testing.T) {
 		ts := newTestServer(t)

